configure.ac

```diff
@@ -418,6 +418,16 @@ if test x$plugin_codemon = xyes; then
   AC_DEFINE_UNQUOTED(ENABLE_PLUGIN_CODEMON, 1, "")
 fi
 
+AC_ARG_ENABLE([plugin_dummy],
+  [AS_HELP_STRING([--enable-plugin-dummy],
+    [Enable dummy plugin for development @<:@no@:>@])],
+  [plugin_dummy="$enableval"],
+  [plugin_dummy=""])
+AM_CONDITIONAL([PLUGIN_DUMMY], [test x$plugin_dummy = xyes])
+if test x$plugin_dummy = xyes; then
+  AC_DEFINE_UNQUOTED(ENABLE_PLUGIN_DUMMY, 1, "")
+fi
```

Comment:
I don't get what this plugin accomplishes. Seems like a hack.

Reply:
It is "a hack", to make development easier (testing something quickly without having to register new plugin in 50 places). That's why it's disabled by default.